config/routes.rb

```diff
@@ -18,6 +18,6 @@
 
   resources :users, except: [:new, :create]
 
-  get '/dashboard', to: 'dashboard#index'
+  get '/dashboard', to: 'users#show'
```

Comment:
Is there any reason in particular that we're using the dashboard route for the user profile page?